cloud/logger/src/index.ts

```diff
@@ -0,0 +1,50 @@
+import Logger from './Logger';
+
+
+export default class LoggerImpl implements Logger {
```

Comment:
Missing info what's the purpose of this class. Just description why we need this instead of explicit BunyanLogger.